terracumber_config/tf_files/SUSEManager-4.2-beta-validation.tf

```diff
@@ -0,0 +1,973 @@
+}
+
+variable "CUCUMBER_BRANCH" {
+  type = "string"
+  default = "master"
+}
+
+variable "CUCUMBER_RESULTS" {
+  type = "string"
+  default = "/root/spacewalk/testsuite"
+}
+
+variable "MAIL_SUBJECT" {
+  type = "string"
+  default = "Results Last Milestone Build Validation $status: $tests scenarios ($failures failed, $errors errors, $skipped skipped, $passed passed)"
+}
+
+variable "MAIL_TEMPLATE" {
+  type = "string"
+  default = "../mail_templates/mail-template-jenkins.txt"
+}
+
+variable "MAIL_SUBJECT_ENV_FAIL" {
+  type = "string"
+  default = "Results Last Milestone Build Validation: Environment setup failed"
+}
+
+variable "MAIL_TEMPLATE_ENV_FAIL" {
+  type = "string"
+  default = "../mail_templates/mail-template-jenkins-env-fail.txt"
+}
+
+variable "MAIL_FROM" {
+  type = "string"
+  default = "galaxy-noise@suse.de"
+}
+
+variable "MAIL_TO" {
+  type = "string"
+  default = "galaxy-noise@suse.de"
+}
+
+// sumaform specific variables
+variable "SCC_USER" {
+  type = "string"
+}
+
+variable "SCC_PASSWORD" {
+  type = "string"
+}
+
+variable "GIT_USER" {
+  type = "string"
+  default = null // Not needed for master, as it is public
+}
+
+variable "GIT_PASSWORD" {
+  type = "string"
+  default = null // Not needed for master, as it is public
+}
+
+provider "libvirt" {
+  uri = "qemu+tcp://arrakis.mgr.prv.suse.net/system"
+}
+
+provider "libvirt" {
+  alias = "caladan"
+  uri = "qemu+tcp://caladan.mgr.prv.suse.net/system"
+}
+
+provider "libvirt" {
+  alias = "giediprime"
+  uri = "qemu+tcp://giediprime.mgr.prv.suse.net/system"
+}
+
+module "base_core" {
+  source = "./modules/base"
+
+  cc_username = var.SCC_USER
+  cc_password = var.SCC_PASSWORD
+  name_prefix = "suma-qam-42-"
+  use_avahi   = false
+  domain      = "mgr.prv.suse.net"
+  images      = [ "sles15sp3o", "opensuse152o" ]
+
+  mirror = "minima-mirror-qam.mgr.prv.suse.net"
+  use_mirror_images = true
+
+  testsuite          = true
+
+  provider_settings = {
+    pool        = "default"
+    bridge      = "br1"
+    additional_network = "192.168.41.0/24"
+  }
+}
+
+module "base_old_sle" {
+  providers = {
+    libvirt = libvirt.caladan
+  }
+
+  source = "./modules/base"
+
+  cc_username = var.SCC_USER
+  cc_password = var.SCC_PASSWORD
+  name_prefix = "suma-qam-42-"
+  use_avahi   = false
+  domain      = "mgr.prv.suse.net"
+  images      = [ "sles11sp4", "sles12sp4o"]
+
+  mirror = "minima-mirror-qam.mgr.prv.suse.net"
+  use_mirror_images = true
+
+  testsuite          = true
+
+  provider_settings = {
+    pool        = "default"
+    bridge      = "br1"
+    additional_network = "192.168.41.0/24"
+  }
+}
+
+module "base_res" {
+  providers = {
+    libvirt = libvirt.caladan
+  }
+
+  source = "./modules/base"
+
+  cc_username = var.SCC_USER
+  cc_password = var.SCC_PASSWORD
+  name_prefix = "suma-qam-42-"
+  use_avahi   = false
+  domain      = "mgr.prv.suse.net"
+  images      = [ "centos6o", "centos7o", "centos8o" ]
+
+  mirror = "minima-mirror-qam2.mgr.prv.suse.net"
```

Comment:
on the long run, we might switch to a more generic name, as this is not QAM :-)

Reply:
agree